Tests/VoiNetworkTests/APIExample/APIExampleRequest.swift

```diff
@@ -28,6 +28,34 @@ enum APIExampleRequest: APIRequest {
         case .exampleRequestWithDictionaryBody(let dictionary): return .jsonFromDictionary(dictionary: dictionary)
         }
     }
+    
+    var parsersMap: [HTTPStatusCode : AnyParser] {
+        get {
+            return [.ok : AnyParser(parser: APIExampleParser()),
+                    .badRequest: AnyParser(parser: APIExampleErrorParser()),
+                    .noContent: AnyParser(parser: APIExampleErrorParser())]
+        }
+    }
+}
+
+final class APIExampleParser: APIParser {
```

Comment:
Would we need to create one new class for each status code we wanna map to and for each request?
could we perhaps use it as a generic parameter to satisfy the associated type?

Reply:
I didn't get it. Could you please provide some small example how it should look?